File: music-android-java/module_ucenter/src/main/java/com/czb/module_ucenter/service/UcenterServiceImpl.java
package com.czb.module_ucenter.service;

import android.content.Context;

import androidx.fragment.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.czb.module_base.RoutePath;
import com.czb.module_base.common.service.ucenter.IUcenterService;
import com.czb.module_ucenter.ui.fragment.UserFragment;


@Route(path = RoutePath.Ucenter.SERVICE_UCENTER)
public class UcenterServiceImpl implements IUcenterService {
    @Override
    public Fragment getFragment() {
        return new UserFragment();
    }

    @Override
    public void init(Context context) {

    }
}

File: music-android/module_ucenter/src/main/java/com/chen/module_ucenter/serivce/UcenterServiceImpl.kt
package com.chen.module_ucenter.serivce

import android.content.Context
import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.chen.module_base.common.constant.RoutePath
import com.chen.module_base.common.service.ucenter.IUcenterService
import com.chen.module_ucenter.fragment.UcenterMainFragment


@Route(path = RoutePath.Ucenter.SERVICE_UCENTER)
class UcenterServiceImpl: IUcenterService {
    override fun getFragment(): Fragment {
        return UcenterMainFragment()
    }

    override fun init(context: Context?) {
    }
}